### PR TITLE
Fix CLI path & Compute node & GitActions

### LINKED ISSUE #7: The no_compute_nodes variable should be "compute nodes per zone" instead of total number of nodes

### LINKED ISSUE #9: Update the alias in module metadata to `cluster`

### LINKED ISSUE #11: Update BIN_DIR path for the module level scripts (wait & delete)
Update BIN_DIR path for the  wait and delete scripts in the modules by setting as environment variable or argument

### PR DESCRIPTION
**Pull request by indirakalagara**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,15 +1,13 @@
 locals {
   cluster_name = var.cluster_name != "" && var.cluster_name != null ? var.cluster_name : "${var.name_prefix}-cluster"
   bin_dir        = module.setup_clis.bin_dir
-  cred_dir       = "${path.cwd}/rosa_user_creds"
-  cred_file_name = "rosa_admin_cred"
-
+  compute_nodes = var.multi-zone-cluster ? (var.no_of_compute_nodes * 3) : var.no_of_compute_nodes
```
Comment (triceam) [inline main.tf:4]: What is the reason for `var.no_of_compute_nodes * 3`?  That seems arbitrary.

File: scripts/delete_cluster.sh [hunk 2]
```diff
@@ -6,18 +6,22 @@ CLUSTER_NAME="$1"
 REGION="$2"
 export AWS_DEFAULT_REGION=${REGION}
 echo "Inputs CLUSTER_NAME  : ${CLUSTER_NAME} , REGION: ${REGION} "
-BIN_DIR=$(cat .bin_dir)
-cat .bin_dir
 export ROSA_TOKEN="$3"
+# BIN_DIR=$(cat .bin_dir)
```
Comment (triceam) [inline scripts/delete_cluster.sh]: Are these commented sections still needed?  If not, maybe remove them to get rid of unnecessary code?

File: scripts/wait-for-cluster-ready.sh [hunk 3]
```diff
@@ -6,9 +6,10 @@ CLUSTER_NAME="$1"
 REGION="$2"
 export AWS_DEFAULT_REGION=${REGION}
 echo "Inputs CLUSTER_NAME  : ${CLUSTER_NAME} , REGION: ${REGION} "
-BIN_DIR=$(cat .bin_dir)
-cat .bin_dir
 
+# if [[ -n "${BIN_DIR}" ]]; then
```
Comment (triceam) [inline scripts/wait-for-cluster-ready.sh]: Same here... if not needed, can we get rid of these commented sections?

File: test/stages/stage1-vpc-subnet.tf [hunk 4]
```diff
@@ -10,7 +10,7 @@ module "dev_pub_subnet" {
   subnet_cidrs       = var.pub_subnet_cidrs
   availability_zones = var.availability_zones
   gateways           = [module.dev_igw.igw_id]
-  acl_rules          = var.acl_rules_pub
+  #acl_rules          = var.acl_rules_pub
```
Comment (triceam) [inline test/stages/stage1-vpc-subnet.tf:13]: Same here... if not needed, can we get rid of these commented sections?

File: test/stages/stage1-vpc-subnet.tf [hunk 5]
```diff
@@ -37,10 +37,9 @@ module "dev_priv_subnet" {
   #availability_zones = ["ap-south-1a","ap-south-1b"]
   subnet_cidrs       = var.priv_subnet_cidrs
   availability_zones = var.availability_zones
-  acl_rules          = var.acl_rules_pri
+  #acl_rules          = var.acl_rules_pri
```
Comment (triceam) [inline test/stages/stage1-vpc-subnet.tf:40]: Same here... if not needed, can we get rid of these commented sections?

File: main.tf (hunk 1 above)
Comment (indirakalagara) [reply main.tf:4]: Here is the issue explain the reason for this change https://github.com/cloud-native-toolkit/terraform-aws-rosa/issues/7. Basically in AWS the input is taken as total number of computed nodes , not per zone. Hence to be inline with other hyperscalers, we are taking input  as  number of computes nodes per zone

File: scripts/delete_cluster.sh (hunk 2 above)
Comment (indirakalagara) [reply scripts/delete_cluster.sh]: Removed comments

File: scripts/wait-for-cluster-ready.sh (hunk 3 above)
Comment (indirakalagara) [reply scripts/wait-for-cluster-ready.sh]: Removed comments

File: test/stages/stage1-vpc-subnet.tf (hunk 4 above)
Comment (indirakalagara) [reply test/stages/stage1-vpc-subnet.tf:13]: Kept it as placeholder for testing with custom acls

File: test/stages/stage1-vpc-subnet.tf (hunk 5 above)
Comment (indirakalagara) [reply test/stages/stage1-vpc-subnet.tf:40]: Kept it as placeholder for testing with custom acls

**Review by seansund — APPROVED** [review]
lgtm

**Pull request merged**